docs/tutorial/install-php-library.txt

```diff
@@ -9,17 +9,13 @@ Prerequisites
 
 The MongoDB PHP Library is a high-level abstraction for the
 MongoDB PHP driver. As such, you must install the `mongodb`
-extension to use the |php-library|:
+extension to use the |php-library|.
 
-.. code-block:: sh
-
-   pecl install mongodb
-   echo "extension=mongodb.so" >> `php --ini | grep "Loaded Configuration" | sed -e "s|.*:\s*||"`
-
-Instructions for installing the `mongodb` extension on HHVM may be
-found in the :php:`Installation with HHVM
-<mongodb.installation.hhvm>` article
-in the driver documentation.
+:php:`Installing the MongoDB PHP Driver <mongodb.installation.php>`
```

Comment:
In [`conf.py`](https://github.com/mongodb/docs-php-library/blob/67875d9409f8774617e1783481068840cc9c4ba5/conf.py#L60), the generated URL will be "http://php.net/%s", which would make this a 404. I'll change to "manual/en/mongo.installation.php".